publish-cli/pom.xml

```diff
@@ -38,11 +39,13 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-actuator</artifactId>
+            <version>2.7.5</version>
```

Comment:
I think we can remove hard coded values, the value is getting inherited from the remrem-parent..

Reply:
replaced by variable